stock-tracker/src/socket/eventTypes.ts

```diff
@@ -0,0 +1,11 @@
+export enum EventTypes {
+  STOCK_DATA = "StockData",
+  SYMBOL_INPUT = "symbol",
+  SEARCH_INPUT = "search",
+  COMPANY_OVERVIEW = "CompanyOverview",
+  LATEST_NEWS = "LatestNews",
+  SUGGESTIONS = "suggestions",
+  CHART_DATA = "ChartData",
+  CHART_TIME = "chartTime",
+  TOP_PEERS = "TopPeers"
+}
```

Comment:
q: Why are you now using `Enums` here?

Reply:
just one enum! It's a more efficient way to use in EventActionInterface event